src/webui/api/torrentscontroller.cpp

```diff
@@ -296,6 +297,7 @@ void TorrentsController::infoAction()
     int limit {params()[u"limit"_s].toInt()};
     int offset {params()[u"offset"_s].toInt()};
     const QStringList hashes {params()[u"hashes"_s].split(u'|', Qt::SkipEmptyParts)};
+    const std::optional<bool> isPrivate = parseBool(params()[u"isPrivate"_s]);
```

Comment:
I would name it `private`.

Reply:
I would strongly expect that we use the same names like the ones within returned dictionary.

Reply:
Same view here. To my understanding, that is as it is currently implemented.

Reply:
>I would strongly expect that we use the same names like the ones within returned dictionary.

OK it is less problematic for this one.

Reply:
> > I would strongly expect that we use the same names like the ones within returned dictionary.
> 
> OK it is less problematic for this one.

But it still needs to be changed. It is called `is_private` in another dictionary:
https://github.com/qbittorrent/qBittorrent/blob/47c05fd3a47295c3dabe241e19d2cfa031ad21c3/src/webui/api/serialize/serialize_torrent.h#L96

So I would change it to `private` in both places.